packages/web/src/components/gcds-link/gcds-link.css

```diff
@@ -14,9 +14,9 @@
   :host .gcds-link {
     cursor: pointer;
     color: var(--gcds-link-default);
-    text-decoration: underline currentColor
-      var(--gcds-link-decoration-thickness);
-    text-underline-offset: var(--gcds-link-underline-offset);
+    text-decoration-style: solid;
+    text-decoration-color: currentColor;
+    text-decoration-thickness: var(--gcds-link-decoration-thickness);
```

Comment:
Seems like this line is getting deleted by accident here - can we put this back in, please?
`text-underline-offset: var(--gcds-link-underline-offset);`

Reply:
Good catch!